app/src/main/java/com/work/restaurant/GoogleMapFragment.kt

```diff
@@ -0,0 +1,120 @@
+package com.work.restaurant
+
+import android.content.Context
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.google.android.gms.maps.CameraUpdateFactory
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.MapView
+import com.google.android.gms.maps.OnMapReadyCallback
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.MarkerOptions
+
+
+class GoogleMapFragment : OnMapReadyCallback, Fragment() {
+
+
+    private lateinit var mapView: MapView
+
+    override fun onMapReady(p0: GoogleMap) {
+
+        val SEOUL = LatLng(37.56, 126.97)
+        val markerOptions = MarkerOptions()
+        markerOptions.position(SEOUL)
+        markerOptions.title("서울")
+        markerOptions.snippet("한국의 수도")
+        p0.addMarker(markerOptions)
+        p0.moveCamera(CameraUpdateFactory.newLatLng(SEOUL))
+        p0.animateCamera(CameraUpdateFactory.zoomTo(10f))
+    }
+
+
+    override fun onAttach(context: Context) {
+        Log.d(fragmentName, "onAttach")
+        super.onAttach(context)
+    }
+
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        Log.d(fragmentName, "onCreate")
+        super.onCreate(savedInstanceState)
+
+
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+
+        val rootView = inflater.inflate(R.layout.google_maps, container, false)
+
+        mapView = rootView.findViewById(R.id.map)
+        mapView.getMapAsync(this)
+
+        return rootView
+    }
+
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        Log.d(fragmentName, "onActivityCreated")
+        super.onActivityCreated(savedInstanceState)
+
+
+        if (mapView != null)
```

Comment:
```suggestion
        if (::mapView.isInitialized)
```
mapView는 not null type이기 때문에 null check를 할 필요가 없어보입니다.

Reply:
5bf0004 수정하였습니다.

Reply:
이런식으로 사용이 되는걸 초기화 된지 확인한다는 의미인건가요?